### DISCUSSION TITLE
Login-free Dashboards

### DISCUSSION THREAD

**Discussion by yityngG** [datalust/seq-tickets#1925] [Help]
Hi, 

I found that for Login-free Dashboards, using the URL, on the top right side, will only have LogOut button and the Events button is missing. 
![image](https://github.com/datalust/seq-tickets/assets/142372835/8a70a0ac-02b5-4450-aded-64be91f8ef05)

Is there any ways to show the Events and Dashboards button on the top right side while using Login-free Dashboards?
![image](https://github.com/datalust/seq-tickets/assets/142372835/25a89c8c-ac09-4d9e-87dd-e8c486551758)

Thanks.

**Comment by nblumhardt** [comment]
Hi @yityngG, thanks for your message.

No, there's currently no mechanism for this. What kind of scenario are you aiming to support?

Best regards,
Nick

**Reply by nblumhardt** [reply]
Thanks for your reply; by "scenario" I meant - is it for a non-interactive heads-up display (e.g. wall-mounted monitor) or for general linking from somewhere else?

**Reply by yityngG** [reply]
General linking from somewhere else

**Reply by nblumhardt** [reply]
Thanks for the info!

Right now the API key-driven login is only really designed around non-interactive use; lots of parts of the app (including Events) will fail if there's no current user.

Definitely something for us to think about, but unfortunately not many options to work around it right now.
